tutorial_guide/deploy_ama_to_linux_arc_server/_index.md

```diff
@@ -0,0 +1,91 @@
+## Overview
+
+This Drop will you guide you through using the Azure Monitor Agent on an existing Linux Arc-enabled server.
+
+## Prerequisites
+
+- this Drop starts at the point where you have already deployed and connected one or more machines to Azure Arc.  If you need an Azure Arc-enabled server, search for other Drops or use a Jumpstart scenario like [Deploy a local Ubuntu server hosted with Vagrant and connect it Azure Arc](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/vagrant/local_vagrant_ubuntu/)
+
+- [Install or update Azure CLI to the latest version](https://learn.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest).
+
+- an existing resource group in Azure
+
+
+## Getting Started
+
+### Deployment
+
+Set the name of your resource group and Arc-enabled server:
+
+```bash
```

Comment:
Maybe add a table explaining each of this parameters and have no default value, just flag name

Reply:
I added a table but also left showing the setting of the parameters to (hopefully) make clear what's expected since the user can match the example parameter names with the screenshots